src/components/NestedRouter.js

```diff
@@ -39,7 +39,10 @@ export function Route({ component: Component, children, ...props }) {
 }
 
 Route.propTypes = {
-  children: PropTypes.node,
+  children: PropTypes.oneOfType([
+    PropTypes.func,
+    PropTypes.object,
```

Comment:
@EthanFreestone is there any reason why you used `object` here instead of `node`?

I wonder if this can be replaced with something like:

 ````js

children: PropTypes.oneOfType([
    PropTypes.arrayOf(PropTypes.node),
    PropTypes.node,
    PropTypes.func
  ])
````

Reply:
Ah, I actually thought I had used node, will change immediately :)